l10n_eu_nace/readme/USAGE.rst

```diff
@@ -0,0 +1,19 @@
+Once this module is installed, you can assign NACE categories to your partners
+by simply adding the corresponding category in the partner's form.
+
+Obtaining updated data
+======================
+The data is imported from the new NACE Rev. 2 classification, which is hosted in
+ShowVoc. This is a new platform that replaces the old Eurostat RAMON service and
+uses SPARQL to retrieve the data.
+
+If you want to import or update the data, you can do so by running the Import
+NACE Wizard. This will search for all active languages in Odoo and import the
+data for each one. If you activate a new language, remember to run this wizard
+again.
```

Comment:
suggestion: I think it'd be helpful to highlight a bit this important section of the docs.

```suggestion
data for each one. **If you activate a new language, remember to run this wizard
again.**
```